microsoft-edge/webview2/how-to/winrt-js-conversion.md

```diff
@@ -0,0 +1,139 @@
+| `Enum` | JavaScript object | An enum type is represented as a JavaScript object.  Each enum value is a `Number` property on the JavaScript object. |
+| `Struct` | JavaScript object | A `Struct` type is converted to a JavaScript object that has property names which correspond to the `Struct` type member names.  This is a two-way conversion. |
+| `Namespace` | JavaScript object | A namespace is represented as a JavaScript object that has a property for each child namespace, enum type, and `RuntimeClass`. |
+| `Class` | JavaScript object | A `RuntimeClass` class is converted to a JavaScript object that has the same methods, properties, and events. |
+| `Interface` | JavaScript object | A `RuntimeClass` interface is converted to a JavaScript object that has the same methods, properties, and events.  There is no support for implementing an interface in JavaScript. |
+| Class static member | JavaScript object property | See below. |
+| Class constructor | JavaScript constructor and function | See below. |
+
+See also:
+* [RuntimeClass class](https://learn.microsoft.com/en-us/cpp/cppcx/wrl/runtimeclass-class)
+
+
+<!-- ------------------------------ -->
+#### Class static members
+
+<!--
+original wording:
+Runtimeclasses with static methods, properties, or events, are represented as properties of their namespace with the static methods, properties, and events as properties on that JavaScript object.
+
+that might mean:
+
+The runtime class that has static properties, methods, or events is a property of the namespace.  Each static property, method, or event is a property of the JavaScript Object.
+
+A `RuntimeClass` class that has static properties, methods, or events is represented as a property of the corresponding JavaScript namespace object.  The static methods, properties, and events are represented as properties on the JavaScript namespace object.
+-->
+A runtime class that has static properties, static methods, or static events is represented as a property of the namespace.  Each static property, method, or event is represented as a property on that JavaScript object.
+
+For example, to call the static method `Windows.Foundation.Uri.EscapeComponent`, call:
+
+```javascript
+`chrome.webview.hostObjects.Windows.Foundation.Uri.EscapeComponent("example");`
+```
+
+The JavaScript namespace object here is `chrome.webview.hostObjects`.
+
+
+<!-- ------------------------------ -->
+#### Class constructors
+
+<!-- orig wording:
+Runtimeclasses with constructors are represented as a JavaScript constructor and function on the namespace object.
+-->
+A constructor for a `RuntimeClass` class is represented as two members on a JavaScript object:
```

Comment:
There's just one property, one thing in JS but it can be called two different ways, either as a function or with new as a constructor:

```suggestion
A constructor for a `RuntimeClass` class is represented as a single property on a JavaScript object that can be called two ways:
```

Reply:
fixed locally